README

```diff
@@ -48,6 +48,13 @@ Git repository for work-in-progress changes is available at:
 <https://github.com/01org/libyami>
 
 
+Test Applications
+-----------------
+To avoid add too many dependency to your project, we split all test applications
+to yami-utils. It include yamidecode/yamiencode and conformance test script.
+You can get it from https://github.com/01org/libyami-utils
+
+
```

Comment:
I don't think we need to explain ourselves for why they are moved to libyami-utils.

How about something like...

    Demos, Examples and Test Applications
    ---------------------------------------------------
    The libyami-utils project provides various example, test and demo
    applications that use libyami.  For more details, please refer to

        https://github.com/01org/libyami-utils

...then we can remove the next section too.